### ISSUE TITLE
Server side message properties: wrong encoding

### ISSUE THREAD

**Issue by fanste** [primefaces/primefaces#7748]
**Describe the defect**
It looks like the newly introduced server side message properties (#6995) do not use the correct encoding. In my case the german umlauts are totally messed up.

Screenshot of the eclipse debug view (class `PropertyResourceBundle` field `lookup`)
![grafik](https://user-images.githubusercontent.com/3531183/130602617-badadcb6-7a2e-4e82-8ef8-59dd69b322d2.png)
![grafik](https://user-images.githubusercontent.com/3531183/130602786-5c3915cd-33d0-46bd-970f-5739ff1feecf.png)

Screenshot of the rendered page
![grafik](https://user-images.githubusercontent.com/3531183/130602920-3a1ac724-8daa-49ad-a213-d68a4a3c313a.png)

If i extract the Message_de.properties file from the JAR and convert the file to ANSI using Notepad++ and repack it into the JAR, everything seems fine:
![grafik](https://user-images.githubusercontent.com/3531183/130602251-91840255-57c8-4cda-a7fd-92e694d4dbc4.png)

The Javadoc of the constructor of `PropertyResourceBundle` contains:
```
The property file read with this constructor must be encoded in ISO-8859-1.
```
Notepad++ tells me, that all property files are UTF-8 (without BOM) (I checked ~10 files).

**Environment:**
 - PF Version: _10.0_
 - JDK 8

**Comment by fanste** [comment]
Is this really a bug in PF 10 or is there anything wrong with our dev environment (all my collegues have the same problem)?

I cannot believe that nobody recognized the wrong encoding while introducing the server side message properties.

_Edit 24.08.2021 13:17:_
Maybe this is the solution why nobody detected that problem:
https://docs.oracle.com/javase/9/intl/internationalization-enhancements-jdk-9.htm#JSINT-GUID-974CF488-23E8-4963-A322-82006A7A14C7

JDK >= 9 introduced UTF-8 support for `PropertyResourceBundle`

**Comment by fanste** [comment]
Reproducer: https://github.com/fanste/primefaces-test/tree/test-7748
![grafik](https://user-images.githubusercontent.com/3531183/130605062-549a8c86-90de-4c4d-971e-610ec7f6cf70.png)

**Comment by stolp** [comment]
Java 8 is not able to work with UTF-8 encoded properties. This has been reported in #7149 and was fixed in the 10.0.3 elite release.

**Comment by fanste** [comment]
Thanks for the hint. Somehow I did not find that issue -> closed
